apps/mobile/src/components/TradeChecklistFlow/components/DateAndTimeFlow/components/ChooseAvailableDaysScreen.tsx

```diff
@@ -45,12 +44,13 @@ const calendarTheme: Theme = {
   selectedDayBackgroundColor: 'transparent',
   textDisabledColor: '#A0A0AB',
 }
+
 const today = DateTime.now().toFormat(reactNativeCalendarsDateFormat)
 
 function ChooseAvailableDaysScreen(): JSX.Element {
   const {t} = useTranslation()
-  const setDateAndTimeScreenState = useSetAtom(
-    setDateAndTimeScreenStateActionAtom
+  const syncAvailableDateTimesWithChat = useSetAtom(
```

Comment:
Do not sync with chat state. Use it's own state and sync with dateTimeFlow state on flow submit